platform.txt

```diff
@@ -5,8 +5,8 @@
 
 # The information about this Arduino compatible environment
 # ---------
-name=XMC Family V2.x
-version=2.0.0
+name=XMC Family V3.x
```

Comment:
```suggestion
name=XMC Family
```
Shall we just remove the version in the name? Is it necessary or was just there for when we have both versions together?

Reply:
makes sense 👍